.stylelintrc.js

```diff
@@ -46,6 +80,8 @@ module.exports = {
           'lost-unit',
           'lost-utility',
           'lost-waffle',
+          'container-name',
+          'container-type',
```

Comment:
Nit: Can we add an empty line to separate these from the group above, so it's clear that the `// TODO(mc, 2018-02-09)` comment doesn't apply to these?

```suggestion

          'container-name',
          'container-type',
```